third_party/openenclave/Makefile

```diff
@@ -96,6 +96,8 @@ endif
 	mkdir -p $(TOP)/build/bin
 	rm -f $(TOP)/build/bin/myst-gdb
 	ln -s $(TOP)/build/openenclave/bin/oegdb $(TOP)/build/bin/myst-gdb
+	# Patch minimum required lldb version from 7 to 8.
+	sed -i 's/-7/-8/g' $(TOP)/build/openenclave/bin/oelldb
```

Comment:
myst_lldb needs to be updated to make sure it is using is using lldb of *greater than*  7. The error message needs updating too.

Reply:
The `sed` command updates the version number in the error message too
> echo "oelldb requires lldb-8 or above."